### ISSUE TITLE
Hotkey mode keeps intermittently turning itself on

### ISSUE THREAD

**Issue by Michiyamenotehifunana** [outer-haven/outer-haven#193]
**Round ID 17**
Step 1: Join the game
Step 2: Type verbs in
Step 3: Spazz the fuck out and do random things because hotkey mode turned itself on. Uncommanded.
Step 4: Hit tab to turn off hotkey mode
Step 5: Do things normally
Step 6: Suddenly spazz out again while typing things because hotkeymode turned itself back on.
Step 7: ?????
Step 8: PROFIT!!!

Hotkey mode appears to be constantly turning itself back on. This is extremely annoying.

**Comment by Kevak** [comment]
I agree it’s annoying. We’ve recieved a good dozen complaints about it. It’s tgshitcode. I’ll be making a revert pr.

**Comment by Kevak** [comment]
Welp. Looks like github literally cannot revert it. Might have to do it entirely fucking manually. This was a bad idea to merge @Partheo I told you it would cause problems. _I was right._

**Comment by praisenarsie** [comment]
the reason why it hasn't been reverted is because there is a revert pr up which is being TESTMERGED.

**Comment by Partheo** [comment]
This is the third time we've gone over this, @Kevak. I know. I know it wasn't a good idea.

What Praisenarsie said is right, the revert is currently testmerged on TG. I'll make a separate branch with the PR reverted in the meantime.
